leakcanary-android-core/src/main/java/leakcanary/internal/InternalLeakCanary.kt

```diff
@@ -143,6 +145,29 @@ internal object InternalLeakCanary : (Application) -> Unit, OnObjectRetainedList
     disableDumpHeapInTests()
   }
 
+  private fun checkAllowInNonDebuggableBuild() {
+    if (AppWatcher.isDebuggableBuild) {
+      return
+    }
+
+    if (application.resources.getBoolean(R.bool.leak_canary_allow_in_non_debuggable_build)) {
+      // AppWatcher is disabled by default for non-debuggable builds, but we have a developer opt-in.
+      AppWatcher.config = AppWatcher.config.copy(enabled = true)
+    } else {
+      throw Error("""
+        LeakCanary in non-debuggable build
+        
+        LeakCanary should only be used in debug builds, but this APK is not marked as debuggable.
+        Please follow the instructions on the "Getting started" page to only include LeakCanary in
+        debug builds: https://square.github.io/leakcanary/getting_started/
+        
+        If you're sure you want to include LeakCanary in a non-debuggable build, you can disable this
+        check by overriding the bool/leak_canary_allow_in_non_debuggable_build resource and setting
+        the value to 'true'.
```

Comment:
I think it would be better to add a corresponding section in the [Code recipes](https://square.github.io/leakcanary/recipes/) and have the error message point to that page with an anchor to a detailed description with xml snippet to copy&paste. Source code of those docs is located in `/leakcanary/docs/recipes.md`

Reply:
A link for just the last part on how to disable the runtime check? Or replace the whole message with a link?